payments/channel.go

```diff
@@ -435,7 +435,7 @@ func (ch *channel) waitForPaychCreateMsg(channelID string, mcid cid.Cid) {
 
 // waitPaychCreateMsg wait for a given confidence index and cleans up if the message failed
 func (ch *channel) waitPaychCreateMsg(channelID string, mcid cid.Cid) error {
-	mwait, err := ch.api.StateWaitMsg(ch.ctx, mcid, uint64(5))
+	mwait, err := ch.api.StateWaitMsg(ch.ctx, mcid, uint64(3))
```

Comment:
no need to be cast in uint64 since it will be guessed by the compiler

Reply:
Lol you're right thanks